CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Changed
+- Upgrade kamu-cli version to 0.XXX.X (ReBAC, TODO)
```

Comment:
TODO: update the text to the actual text when PR is merged
- https://github.com/kamu-data/kamu-cli/pull/761

Reply:
Done